README.md

```diff
@@ -1 +1,61 @@
-# Webpack
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# T0-DO-LIST
+
+> This is my second project in Module 2 where I build a simple to do list application.
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+This is an example of how to list things you need to use the software and how to install them.
+
+- Web Browser
+- VS Code
+
+### Install
+
+1. Clone the repo or download the repo files
+
+> git clone git@github.com:ramrod433/To-Do-List.git
+
+2. Open folder or zip in VS Code
+
+3. Run the HTML file (index.html) on Browser locally or with Live Server
+
```

Comment:
- [ ] [OPTIONAL] Your project meets the necessary requirements, however, kindly ensure you add a set of instructions that will help users run the project locally: 
```
run npm i
```
```
run npm start
```

Reply:
Yes please. Thank you